objectivec/GPBBootstrap.h

```diff
@@ -28,44 +28,58 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-// The Objective C runtime has complete enough info that most protos don’t end
-// up using this, so leaving it on is no cost or very little cost.  If you
-// happen to see it causing bloat, this is the way to disable it. If you do
-// need to disable it, try only disabling it for Release builds as having
-// full TextFormat can be useful for debugging.
+/**
+The Objective C runtime has complete enough info that most protos don’t end
+up using this, so leaving it on is no cost or very little cost.  If you
+happen to see it causing bloat, this is the way to disable it. If you do
+need to disable it, try only disabling it for Release builds as having
+full TextFormat can be useful for debugging.
+*/
```

Comment:
Looking at the raw headers, this formatting feels a little odd to me

```Objective-C
/**
 *  The Objective C runtime has complete enough info that most protos don’t end 
 *  up using this, so leaving it on is no cost or very little cost.  If you
 *  happen to see it causing bloat, this is the way to disable it. If you do
 *  need to disable it, try only disabling it for Release builds as having
 *  full TextFormat can be useful for debugging.
 **/
```
Seems like it flows a little better and doesn't mix with the code as much.  Thoughts?